docs/api.md

```diff
@@ -304,6 +344,7 @@ Settings related to deploying an image to the host.
 * *id* -- The unique identifier for the service in the underlying
   provisioning tool.
 * *image* -- The image most recently provisioned to the host.
+* *raid* -- The list of hardware and software RAID volumes recently set.
```

Comment:
```suggestion
* *raid* -- The list of hardware or software RAID volumes recently set.
```

You can't set both at the same time. We might want to mention that more prominently as well.

Reply:
oh yes. thanks